recipe/meta.yaml

```diff
@@ -33,11 +31,48 @@ requirements:
     - pytoml
     - pip
 
-test:
-  imports:
-    - flit
-  commands:
-    - flit --version
+outputs:
+  - name: flit-core
+    script: install-core.sh  # [not win]
+    script: install-core.bat  # [win]
+    requirements:
+      host:
+        - python
+        - pip
+        - pytoml
+      run:
+        - python
+        - pytoml
+      test:
```

Comment:
The indent here looks wrong to me. This should not be in `requirements` I think.